### PR TITLE
After the changes to the readme - the exploration of merging and pull req…

### PR DESCRIPTION
**Pull request by ajmpawlik**
…uests

#1 

Description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # TEAM SCORCHIO: MAKERS BnB
 ----------
-
+THIS IS A CHANGE IN THE README
 ## HOW TO USE
```
Comment (rachjgriff) [inline README.md]: What a silly thing to add to the readme!!!!!

File: new_file.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+"A content"
```
Comment (rachjgriff) [inline new_file.rb]: What a pointless file!

File: README.md (hunk 1 above)
Comment (ajmpawlik) [reply README.md]: :P

**Comment by ajmpawlik** [comment]
This is the first merging exercise.

**Pull request merged**